### ISSUE TITLE
Kubernetes monitoring dashboard does not show pods metrics

### ISSUE THREAD

**Issue by muzammil360** [felipeagger/k8s-all-in-one#3]
[Kubernetes monitoring dashboard](https://grafana.com/grafana/dashboards/12740-kubernetes-monitoring/) works great in general but it does not show pod related information. It appears that there is no label `name` associated with the metric which starts with `k8s_`. As a result no time series data is returned. Can you share why are we expecting the pod name to start with `k8s_`? 

 

<img width="1039" alt="image" src="https://github.com/felipeagger/k8s-all-in-one/assets/20393633/028c0df4-4b93-4a6c-8cae-6110424ead4b">

**Comment by XavierFilipe** [comment]
@muzammil360 you are right, i think this is dsnt make sense... can you make a PR to fix?

**Comment by webbrain-one** [comment]
Thanks for flagging this! I opened [PR #4](https://github.com/felipeagger/k8s-all-in-one/pull/4) to fix the pod metrics issue. It adjusts the dashboard queries to match the metric names/labels actually exposed by `k8s-all-in-one`, so the pod-related panels should start returning data again.
